x11-packages/adwaita-icon-theme/build.sh

```diff
@@ -3,9 +3,10 @@ TERMUX_PKG_DESCRIPTION="GNOME standard icons"
 TERMUX_PKG_LICENSE="LGPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="46.2"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://download.gnome.org/sources/adwaita-icon-theme/${TERMUX_PKG_VERSION%.*}/adwaita-icon-theme-${TERMUX_PKG_VERSION}.tar.xz
 TERMUX_PKG_SHA256=beb126b9429339ba762e0818d5e73b2c46f444975bf80076366eae2d0f96b5cb
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="hicolor-icon-theme, librsvg"
+TERMUX_PKG_DEPENDS="adwaita-icon-theme-legacy, hicolor-icon-theme"
```

Comment:
huh does adwaita no longer depend on `librsvg`?

Reply:
Seems to check out, alright good to merge then,

Reply:
librsvg is not used anywhere in the adwaita-icon-theme project. librsvg was mentioned in this 3 years old commit https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/commit/f0b40c37a5073449446d5a5c6d34090eade3486c and is still present in GitLab CI file :man_facepalming:

Reply:
Leave it to GNOME to invent phantom dependencies...

Reply:
Probably, it is used to convert SVG files to PNG.